frontend/client/src/pages/GoodsDetail/GoodsImageSection/ImageList/ImageItem/ImageItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  index: number;
+  src: string;
+  isActivated: boolean;
+  setActive: React.Dispatch<React.SetStateAction<number>>;
+}
+
+const ImageItem: React.FC<Props> = ({ index, src, isActivated, setActive }: Props) => {
+  const onMouseOver = useCallback((e) => {
+    setActive(index);
+  }, []);
+  return <ImageItemContainer src={src} isActivated={isActivated} onMouseOver={onMouseOver} />;
+};
+
+const ImageItemContainer = styled.img<{ isActivated: boolean }>`
+  border: 1px solid ${(props) => (props.isActivated && 'red') || 'transparent'};
```

Comment:
삼항연산자가 더 깔끔하지 않을까요?!
```suggestion
  border: 1px solid ${(props) => props.isActivated ? 'red' : 'transparent'};
```

Reply:
오... 그렇네요!  
그런데 prettier로 인해 소괄호는 기본적으로 추가되네요 :(  

바로 수정했습니다! 👍